olexandr.vakulinskiy/HW2/Task1/Tamagotchi.rb

```diff
@@ -0,0 +1,109 @@
+class Tamagotchi
+
+  attr_accessor :name, :health, :hunger, :thirst, :fatigue
+
+  def initialize(name)
+    @name = name
+    @health = 10
+    @hunger = 2
+    @thirst = 2
+    @fatigue = 0
+  end
+
+  def check_health
+    if @hunger == 0 || @thirst == 0
+      @health += 1
+    elsif @hunger >= 7 || @thirst >= 7
+      @health -= 1
+    end
+    if @fatigue == 10
+      @health = 0
+    elsif  @fatigue >=7
+      @health -= 2
+    end
+
+  end
+  def feed
```

Comment:
There should be a separation empty line between methods. Please, read more [here](https://github.com/rubocop-hq/ruby-style-guide#empty-lines-between-methods)